go/README.md

```diff
@@ -0,0 +1,55 @@
+# Go Target for Snowball
+
+The initial implementation was built as a port of the Rust target.  The initial focus has been on getting it to function, and making it work correctly.  No attempt has been made to beautify the implementation, generate code, or address performance issues.
+
+## Usage
+
+Until this PR is accepted, don't use `go get` to get the runtime library in place.  Instead, clone the mschoch repo into `$GOPATH/src/github.com/snowballstem/snowball`.
```

Comment:
This note doesn't make sense to merge into the main repo - it would be wrong the moment it was merged.